drivers/usb/device/usb_dc_stm32.c

```diff
@@ -363,7 +371,9 @@ static int usb_dc_stm32_phy_specific_clock_enable(const struct device *const clk
 		LOG_ERR("Unable to enable USB clock");
 		return -EIO;
 	}
-
+#if defined(CONFIG_SOC_SERIES_STM32H7RSX) && DT_HAS_COMPAT_STATUS_OKAY(st_stm32_otghs)
+	/* USB OTG HS receives the 60MHz from RCC : no domain clock expected */
+#else
```

Comment:
Same comment as for **udc_stm32.c**: handle that from **drivers/usb/udc/Kconfig.stm32:**?